ansible/roles/adm/tasks/main.yml

```diff
@@ -4,6 +4,10 @@
 # point of truth for the keys, unknown keys are dropped, that's why
 # `authorized_key` ansible module is not used.
 
+- name: Run "apt-get update"
+  apt:
+    update_cache: yes
+
```

Comment:
> how do we manage updates

We manage it with power of `nop`. I mean, we don't. :-( I add another bullet to #101